src/styles/blocks/card.scss

```diff
@@ -0,0 +1,100 @@
+.card {
+  font-family: "Roboto", sans-serif;
```

Comment:
Set the main font family to the body element instead.
Also, [STYLES] - Remember to use fallback fonts - alternative font-family in case the main one doesn't work [like this](https://www.w3schools.com/cssref/pr_font_font-family.asp)
![image](https://github.com/mate-academy/layout_catalog/assets/63243136/546dbd81-ccdf-4fca-844f-3d77c33b5a84)